### ISSUE TITLE
MMKV.getDefaultMMKV occasional crash

### ISSUE THREAD

**Issue by heigoo** [Tencent/MMKV#618]
### The language of MMKV

Java

### The version of MMKV

implementation 'com.tencent:mmkv-static:1.2.7'

### The platform of MMKV

 Android

### The installation of MMKV

Maven

### What's the issue?

java stacktrace:
  at com.tencent.mmkv.MMKV.getDefaultMMKV(Native method)
  at com.tencent.mmkv.MMKV.defaultMMKV(MMKV.java:271)
  at com.sdses.acc.ui.WelcomeActivity.getStoreVersion(WelcomeActivity.java:845)

java code: WelcomeActivity.java:845
` private int getStoreVersion() { return MMKV.defaultMMKV().getInt(Constants.DBVersion, -1);}`

### What's the log of MMKV when that happened?
[native.txt](https://github.com/Tencent/MMKV/files/5807523/native.txt)

**Comment by lingol** [comment]
Assuming your ABI is armeabi, here are the symbolized call stacks:
>std::__ndk1::__bucket_list_deallocator<std::__ndk1::allocator<std::__ndk1::__hash_node_base<std::__ndk1::__hash_node<std::__ndk1::__hash_value_type<std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> >, MMKV*>, void*>*>*> >::size() const
/Users/lingol/Library/Android/sdk/android-ndk-r16b/sources/cxx-stl/llvm-libc++/include/__hash_table:815
std::__ndk1::unordered_map<std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> >, MMKV*, std::__ndk1::hash<std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > >, std::__ndk1::equal_to<std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > >, std::__ndk1::allocator<std::__ndk1::pair<std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const, MMKV*> > >::find(std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> > const&)
/Users/lingol/Library/Android/sdk/android-ndk-r16b/sources/cxx-stl/llvm-libc++/include/unordered_map:1102
MMKV::defaultMMKV(MMKVMode, std::__ndk1::basic_string<char, std::__ndk1::char_traits<char>, std::__ndk1::allocator<char> >*)
/Users/lingol/Developer/mmkv/Core/MMKV.cpp:146
mmkv::getDefaultMMKV(_JNIEnv*, _jobject*, int, _jstring*)
/Users/lingol/Developer/mmkv/Android/MMKV/mmkv/.cxx/cmake/StaticCppRelease/armeabi/../../../../src/main/cpp/native-bridge.cpp:296

In other words, it crashed inside C++ STL library, aka `std::unordered_map.find()`. Nothing we can do about it.
